src/components/gui/gui.jsx

```diff
@@ -257,6 +277,15 @@ GUIComponent.defaultProps = {
         host: null,
         visible: false
     },
-    basePath: './'
+    basePath: './',
+    stageSizeMode: STAGE_SIZES.large
 };
-export default injectIntl(GUIComponent);
+
+const mapStateToProps = state => ({
+    // This is the button's mode, as opposed to the actual current state
+    stageSizeMode: state.scratchGui.stageSize.stageSize
```

Comment:
I think we're conflating two distinct things here. There's the mode, large and small, and there's the display size, which is responsive to screen width.  The display size doesn't change in small mode, but it adjusts in large mode.  We'd never want the `stageSizeMode === STAGE_SIZES.largeConstrained` but that seems like it could be a possibility since it's choosing from the same enumeration.  I think we want to distinguish these things.